fem/sidredatacollection.cpp

```diff
@@ -489,6 +504,73 @@ createMeshBlueprintTopologies(bool hasBP, const std::string& mesh_name)
 }
 
 // private method
+#ifdef MFEM_USE_MPI
+void SidreDataCollection::createMeshBlueprintAdjacencies(bool hasBP)
+{
+   ParMesh *pmesh = dynamic_cast<ParMesh*>(mesh);
+
+   const int GRP_SZ = 35;
+   char group_str[GRP_SZ];
+
+   // TODO(JRC): Separate this out into group hierarchy setup and data allocation
+   // stages like all of the other "createMeshBlueprint*" functions.
```

Comment:
I don't know whether or not this is something we want to worry about for this initial implementation, but the current version of `SidreDataCollection::createMeshBlueprintAdjacencies` doesn't perform separate data allocation and data initialization, which means it doesn't work optimally in cases where a collection is filled with new data (i.e. when `hasBP = false` in `SidreDataCollection::SetMesh`).  If this lack of functionality is acceptable for now, I think it would be a good idea to add an issue as a reminder to revise this function at some point in the future.